frontend/plugins/frontline_ui/src/modules/channels/components/settings/channels-list/ChannelsColumns.tsx

```
@@ -0,0 +1,86 @@
+import { useNavigate } from 'react-router-dom';
+
+export const channelsColumns: ColumnDef<IChannel>[] = [
+  channelsMoreColumn,
+  {
+    id: 'name',
+    accessorKey: 'name',
+    header: 'title',
+    cell: ({ cell }) => {
+      const { icon, _id } = cell.row.original;
+
+      const navigate = useNavigate();
```

ESLint

[error] React Hook "useNavigate" is called in function "cell" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)